tests/pykafka/test_balancedconsumer.py

```diff
@@ -4,11 +4,10 @@
 import unittest2
```

Comment:
These tests should actually go in `test_simpleconsumer`, since the change was made there.